oauth2client/client.py

```diff
@@ -2084,13 +2086,14 @@ def step2_exchange(self, code=None, http=None, device_flow_info=None):
                 token_response=d, scopes=self.scope,
                 token_info_uri=self.token_info_uri)
         else:
-            logger.info('Failed to retrieve access token: %s', content)
+            logger.info('Failed to retrieve access token: %s', response.data)
             if 'error' in d:
                 # you never know what those providers got to say
                 error_msg = (str(d['error']) +
                              str(d.get('error_description', '')))
             else:
-                error_msg = 'Invalid response: {0}.'.format(str(resp.status))
+                error_msg = 'Invalid response: {0}.'.format(
+                    str(response.status))
```

Comment:
I would just rather not fix this right now. This PR has changed enough without adding this in. We can file an issue, if you want.

(I know we generally have a rule of thumb of sprucing up every block of code we touch, but these transport changes are going to touch a majority of this project and I want to be focused on just making transport-related changes. We'll hopefully be combing over every module during the great refactor)